README.md

```diff
@@ -1 +1,27 @@
-Poort8.Ishare.Service
\ No newline at end of file
+Poort8.Ishare.Service
+
```

Comment:
Maybe add an opening paragraph, referring to the ishare endpoints. (Like: https://github.com/POORT8/Poort8.Ishare.Common#poort8isharecommon)